docs/foundations/spacings.md

```diff
@@ -1 +1,15 @@
 # Spacings
+
+Space are used to create distance around element or separating objects. When translate to CSS properties, these are padding, margin, and aboslute positions. 
+
+General rule of thumb, less spacing are use between small components, or components that have a close functional relationship.
+
+<SpacingGroup>
+  <Spacing name="space-xs">between icon + text</Spacing>
+  <Spacing name="space-base">between buttons</Spacing>
+  <Spacing name="space-sm">dropdown, drawer, input field spacing</Spacing>
+  <Spacing name="space-md">between doc cards or modal sections</Spacing>
+  <Spacing name="space-lg" />
+  <Spacing name="space-xl">between page sections in repo page, margin for containers</Spacing>
+</SpacingGroup>
```

Comment:
This will allow @shujuuu @emiliyatrakhtenberg edit the markdown files easily, change descriptions, add images or anything else with a simple Pull Request